install/kubernetes/helm/istio/charts/pilot/templates/crds.yaml

```diff
@@ -1,8 +1,9 @@
-# Pilot CRDs
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   name: destinationpolicies.config.istio.io
+  labels:
```

Comment:
We want istio to support manually injected pods - and those will use some well-known names.

Please remove the customization of pilot name ( and similar ones ). 

Would be good to have a values.xml or other doc with all the knobs - some may make 
sense, but we need to figure out which are important to the user and which can
be kept for advanced users ( or removed ).

Reply:
Don't get what you mean with the customization of pilot names. Those CRD were already defined before with those names.